01-deny-all-traffic-to-an-application.md

```diff
@@ -17,11 +17,11 @@ application, selected using Pod Selectors.
 
 Run a nginx Pod with labels `app=web`  and expose it at port 80:
 
-    kubectl run --generator=run-pod/v1 web --image=nginx --labels app=web --expose --port 80
+    kubectl run web --image=nginx --labels app=web --expose --port 80
```

Comment:
Let's standarize how we passe labels to kubectl according to the help page of kubectl they should look like

--labels="key=value"

So for this on, please replace with --labels="app=web"

Reply:
Done